### ISSUE TITLE
build differs from master

### ISSUE THREAD

**Issue by m5o** [thomaspark/bootswatch#743]
Hey @thomaspark 

When I run `grunt swatch` on the latest master branch, a lot of files get changes.
Shouldn't these files be equal with the committed files? Is there a random salt added?

### reproduce with

    $ git fetch upstream
    $ git merge upstream/master
    $ grunt swatch

```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   dist/cyborg/bootstrap.min.css
	modified:   dist/darkly/bootstrap.min.css
	modified:   dist/flatly/bootstrap.min.css
	modified:   dist/litera/bootstrap.css
	modified:   dist/litera/bootstrap.min.css
	modified:   dist/lux/bootstrap.min.css
	modified:   dist/materia/bootstrap.min.css
	modified:   dist/pulse/bootstrap.min.css
	modified:   dist/slate/bootstrap.css
	modified:   dist/slate/bootstrap.min.css
	modified:   dist/solar/bootstrap.min.css
	modified:   dist/superhero/bootstrap.min.css
	modified:   dist/yeti/bootstrap.min.css
	modified:   docs/4/cyborg/bootstrap.min.css
	modified:   docs/4/darkly/bootstrap.min.css
	modified:   docs/4/flatly/bootstrap.min.css
	modified:   docs/4/litera/bootstrap.css
	modified:   docs/4/litera/bootstrap.min.css
	modified:   docs/4/lux/bootstrap.min.css
	modified:   docs/4/materia/bootstrap.min.css
	modified:   docs/4/pulse/bootstrap.min.css
	modified:   docs/4/slate/bootstrap.css
	modified:   docs/4/slate/bootstrap.min.css
	modified:   docs/4/solar/bootstrap.min.css
	modified:   docs/4/superhero/bootstrap.min.css
	modified:   docs/4/yeti/bootstrap.min.css
```

**Comment by thomaspark** [comment]
Hey @m5o, I believe I merged a pull request that changed some sass files, but didn't subsequently rebuild and commit the new css. Let me try doing that now.

But that shouldn't have touched so many themes. I suspect there is something else going on too, so let's keep this issue open and investigate.

**Comment by thomaspark** [comment]
I just rebuilt, can you run `grunt swatch` and check that nothing gets changed?

**Comment by m5o** [comment]
> I just rebuilt, can you run `grunt swatch` and check that nothing gets changed?

now it's equal, thank you

fixes #743

**Comment by m5o** [comment]
Mmh, the first run was equal and it looks like the problem is gone. But something is strange…

For example: After a change to the cosmo `dist/cosmo/_variables.scss` i've run `grunt swatch:cosmo` also the yeti file get an update. This is not correct, isn't it?

```
$ git diff
diff --git a/dist/cosmo/_variables.scss b/dist/cosmo/_variables.scss
index d750f89e..c51a125f 100755
--- a/dist/cosmo/_variables.scss
+++ b/dist/cosmo/_variables.scss
@@ -37,15 +37,17 @@ $danger:        $red !default;
 $light:         $gray-100 !default;
 $dark:          $gray-800 !default;

+$enable-rounded: false !default;
+
 // Body

 $body-color:    $gray-800 !default;

 // Components

-$border-radius:          0 !default;
-$border-radius-lg:       0 !default;
-$border-radius-sm:       0 !default;
+//$border-radius:          0 !default;
+//$border-radius-lg:       0 !default;
+//$border-radius-sm:       0 !default;

 // Fonts
```

```
$ git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   dist/cosmo/_variables.scss
	modified:   dist/cosmo/bootstrap.css
	modified:   dist/cosmo/bootstrap.min.css
	modified:   dist/yeti/bootstrap.min.css
	modified:   docs/4/cosmo/_variables.scss
	modified:   docs/4/cosmo/bootstrap.css
	modified:   docs/4/cosmo/bootstrap.min.css
	modified:   docs/4/yeti/bootstrap.min.css
```

**Comment by thomaspark** [comment]
I'm noticing a similar thing. Only affects the minified css, but for an untouched theme. Some kind of interaction between the minified css and the postcss script that's applied afterwards?

One issue is that with each change, the postcss script is run on all themes instead of just the modified one. This should be changed.

But even so, this shouldn't happen. It seems to be going back and forth between this:

```css
@supports (position: -webkit-sticky) or (position: sticky){.sticky-top{position:-webkit-sticky;position:sticky;top:0;z-index:1020}}.sr-only{position:absolute;width:1px;height:1px;padding:0;overflow:hidden;clip:rect(0, 0, 0, 0);
```

and

```css
@supports ((position: -webkit-sticky) or (position: sticky)){.sticky-top{position:-webkit-sticky;position:sticky;top:0;z-index:1020}}.sr-only{position:absolute;width:1px;height:1px;padding:0;overflow:hidden;clip:rect(0, 0, 0, 0);
```

Basically adding/removing the extra parens. Maybe minify after postcss using a postcss-based minifier? Ideas?

**Comment by thomaspark** [comment]
Hey @m5o, this should be fixed in 517bddd11579b8aa405f30f5959c4aa379d88238. The issue was that both postcss -> minify and minify -> postcss were happening.
